unit-10/main.js

```diff
@@ -0,0 +1,60 @@
+// exercise 1
+function getMultiplicationTable(number) {
+  for (let i = 1; i <= 10; i++) console.log(number + ' * ' + i + ' = ' + number*i)
+}
+
+// exercise 2
+function printEvenNumber(number) {
+  const error = 'yêu cầu nhập vào số nguyên dương trong khoảng từ 1 đến 30'
+  if (typeof number != 'number' || number < 1 || number > 30) return error
```

Comment:
```suggestion
  const error = 'yêu cầu nhập vào số nguyên dương trong khoảng từ 1 đến 30'

  if (typeof number !== 'number' || number < 1 || number > 30) return error
```
- Viết tách nhau ra
- Dùng `!==` thay cho `!=`